config/_default/menus/menus.en.yaml

```diff
@@ -2413,36 +2413,71 @@ main:
     parent: dbm_setup_sql_server
     identifier: dbm_troubleshooting_sql_server
     weight: 107
+  - name: Setting up Oracle
```

Comment:
I noticed we capitalize the u in "up" here for the child pages, e.g. https://docs-staging.datadoghq.com/deforest/dbm-oracle-beta/database_monitoring/setup_oracle/selfhosted/

Reply:
Yes! Good catch. I'll fix it.